### PR TITLE
Adds one room to the backstreets pool

### PR DESCRIPTION
**Pull request by JustTouhou**
## About The Pull Request
Additional room for fixers to explore. Includes a forge and some weapon mods.

## Why It's Good For The Game
Something good for workshop fixers to find or be sold to.

## Changelog
:cl:
add: new map
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Small thing, forgot the code snippet needed.

File: ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm [hunk 1]
```diff
@@ -14,3 +14,9 @@
 	template_height = 6
 	stock = 3
 	room_type = "backstreets_medium_north"
+
+ /datum/map_template/random_room/backstreets/medium_north/workshop_mn
+    name = "Workshop - Medium North"
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm:19]: Got an extra space in here, just make it one tab

File: ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm [hunk 2]
```diff
@@ -14,3 +14,9 @@
 	template_height = 6
 	stock = 3
 	room_type = "backstreets_medium_north"
+
+ /datum/map_template/random_room/backstreets/medium_north/workshop_mn
```
Comment (Kitsunemitsu) [inline ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm:18]: Remove the space at the back here, DM is based heavily on tabs, and spaces screw it up

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]

File: ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm (hunk 2 above)
Comment (JustTouhou) [reply ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm]: found the space, removed

File: ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm (hunk 1 above)
Comment (JustTouhou) [reply ModularTegustation/tegu_items/backstreets/random_rooms/medium/north.dm]: think I got it, please check

**Review by Kitsunemitsu — COMMENTED** [review]
Great first PR!
Everything looks great

**Review by Kitsunemitsu — APPROVED** [review]

**Pull request merged**